### PR TITLE
get some fix from jubianchi about translation + get last version from…

### PR DESCRIPTION
**Pull request by Grummfy**
… crowdin

### CODE REVIEW

**Comment by marmotz** [comment]
Good job !

**Comment by Grummfy** [comment]
I have done nothing just take stuff from crowdin + update last version on crowdin ;)

File: source/en/asserters/stream.inc.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ isRead
        )
        ->if(file_get_contents($streamController->getPath()))
        ->stream($streamController)
-           ->isRead() // passes
+           ->isRead() // passe
```
Comment (agallou) [inline source/en/asserters/stream.inc.rst]: "passe" on the english version ?

Comment (Grummfy) [inline source/en/asserters/stream.inc.rst]: je corrige sur crowdin, au prochain changement cela sera corrigé

**Pull request merged**